CODE_OF_CONDUCT.md

```diff
@@ -1,100 +1,5 @@
-The SkyPy Code of Conduct
-=========================
```

Comment:
Do we need to get rid of the heading?

Reply:
We should get rid of it, this document is *not* the code of conduct, just a place with a link to the *actual* code of conduct